supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -29,20 +29,20 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 // .Other updates
 // * TODO
 
-// [[august-2023]]
-// == August 2023
-//
-// // TODO: Didn't mention anything in July for the IBM entries that were removed; they've been added back with "do not use". Show these in August's section
-//
-// .Glossary entries
-// * xref:ibm-eserver-system-p[IBM eServer System p]: Updated to "do not use"; use "IBM Power" instead
-// * xref:ibm-s-390[IBM S/390]: Updated to "do not use"; use "IBM Z" instead
-// * xref:iseries[ISeries]: Updated to "do not use"; use "IBM Power" instead
-// * xref:pseries[pSeries]: Updated to "do not use"; use "IBM Power" instead
-//
-// .Style guidance
-// * TODO
-//
+[[august-2023]]
+== August 2023
+
+.Glossary entries
+* xref:ibm-eserver-system-p[IBM eServer System p]: Updated to "do not use"; use "IBM Power" instead
+* xref:ibm-s-390[IBM S/390]: Updated to "do not use"; use "IBM Z" instead
+* xref:iseries[ISeries]: Updated to "do not use"; use "IBM Power" instead
+* xref:pseries[pSeries]: Updated to "do not use"; use "IBM Power" instead
+* Removed _agnostic_, _client side_, _client-side_, _cloud_, _colocate_, _data center_, and _devops_ glossary entries.
```

Comment:
See January's section to see how we did past removals of glossary terms.

So I would put each one of these in the list by itself, with the reason. Something like the following, for each:

```suggestion
agnostic: Removed glossary entry because it already exists in the _IBM Style_ guide
```

Reply:
Fixed